app/views/organisations/new.html.erb

```diff
@@ -6,16 +6,35 @@
 <%= form_for @organisation, url: organisations_path do |form| %>
   <div class="govuk-form-group <%= field_error(@organisation, "name") %>">
     <%= form.label :name, "Organisation name", class: "govuk-label" %>
-    <div class="govuk-hint">If your organisation name does not appear below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
+    <div class="govuk-hint">If your organisation name does not appear
+      below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
     <%= form.select :name, options_for_select(@register_organisations << "", ""), {}, class: "govuk-select" %>
   </div>
 
   <div class="govuk-form-group <%= field_error(@organisation, "service_email") %>">
-    <%= form.label :service_email, "Service email", class: "govuk-label" %>
+    <% if @organisation.errors.attribute_names.include?(:name) %>
+      <span class="govuk-error-message" id="location_error">
+        <span class="govuk-visually-hidden">Error:</span><%= form.label(@organisation.errors.full_messages_for(:name).first) %>
```

Comment:
The `form.label` is missing the target, so the HTML ends up like:

<img width="718" alt="Screenshot 2021-09-07 at 06 18 27" src="https://user-images.githubusercontent.com/825374/132328591-6296cf5b-cb73-4e0d-ac4f-c6210d84be4f.png">

It needs `:service_email` as first param...

That said, I don't think we should be using labels for errors... a form field should have a single label.

Reply:
Label removed.